viewer/static/css/experiment-metadata.styl

```diff
@@ -0,0 +1,33 @@
+@import 'lib';
+
+$grid-padding = $min-padding;
+
+#experiment-metadata {
+  display: grid;
```

Comment:
It's fun that we can use these for internal projects - still wouldn't recommend for anything public facing.